### ISSUE TITLE
Site Editing: Header and Footer template areas stuck in perpetual loading mode

### ISSUE THREAD

**Issue by aurooba** [WordPress/gutenberg#28158]
**Describe the bug**
When editing any template, the header and footer stay in perpetual load mode and prevent any editing of those areas.

*This bug is a result of responding to [FSE Program Testing Call #1: Template Editing](https://make.wordpress.org/test/2020/12/23/fse-program-testing-call-1-template-editing/)*

**To reproduce**
1. Make sure you have the Gutenberg Plugin installed and are using TT1 Blocks Theme
2. Go to Site Editor
2. See the loading circles in lieu of the header and footer:
<img width="1699" alt="Screen Shot 2021-01-12 at 8 28 49 PM" src="https://user-images.githubusercontent.com/6925260/104403467-d029b300-5515-11eb-8c42-da1b3488bcf6.png">

**Expected behavior**
To be able to see an editable block-version of the header and footer.

**Editor version (please complete the following information):**
- WordPress version: 5.6
- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? *Gutenberg plugin installed*
- If the Gutenberg plugin is installed, which version is it? *9.7.1*

**Desktop (please complete the following information):**
 - OS: Mac OS 11.1
 - Browser:Google Chrome
 - Version: 87.0.4280.88

**Additional context**
- Template Editing otherwise seems to be working, although it's not particularly obvious that you are creating a change that will be implemented site-wide.

**Comment by annezazu** [comment]
@aurooba thanks for filing this! This sounds very similar to what @bph ran into [here](https://wordpress.slack.com/archives/C015GUFFC00/p1609306215138000). It might be related to this: https://github.com/WordPress/gutenberg/issues/28022 From what I can see in slack, what resolved this was copying the individual themes into the /themes/ folder. Can you give that a try? 

cc @carolinan in case you might have any ideas :)

**Comment by annezazu** [comment]
Looks like this might be in place to fix this :) https://github.com/WordPress/gutenberg/pull/28088

**Comment by chthonic-ds** [comment]
"This bug is a result of responding to FSE Program Testing Call #1: Template Editing"

I encountered this issue for the same reason. Have tested the solution above and can confirm ensuring a theme is a direct child of the `themes` folder will allow the header and footer to load correctly.

**Comment by nvjacobo** [comment]
Hi, 

This issue also happened to me in my Wordpress and Gutenberg installation for FSE Program Testing Call (Wordpress 5.6 - Gutenberg 9.7.0).

My directory structure was: 

```
theme-experiments
twentynineteen
twentytwenty
twentytwentyone
```
Today I applied @annezazu recommendation. It works well on my installation now. My themes directory structure is:

```
tt1-blocks
twentynineteen
twentytwenty
twentytwentyone
```

Update Gutenberg to 9.7.2

I updated to latest version Gutenberg 9.7.2 with fix #28088 and this issue not happen with my first directory structure

```
theme-experiments
twentynineteen
twentytwenty
twentytwentyone
```

Thanks, Jacobo
